docfiles/docs.js

```diff
@@ -47,8 +54,21 @@ function searchSubmit(form) {
 }
 
 function setupSidebar() {
+    let togglesidebar = document.getElementById("togglesidebar");
+    togglesidebar.onkeydown = handleEnterKey;
```

Comment:
^^ That being said, we have JQuery today and the reason we have it is because of semantic ui. Once we switch to a fully react semantic, we probably will have to refactor this code anyway.